build.gradle

```diff
@@ -14,10 +14,13 @@ version = '2.0.2'
 
 description = 'A Java implementation of the IAB Consent String spec.'
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
 
 dependencies {
+    implementation("com.google.guava:guava:20.0")  {
```

Comment:
I would be really nice to have this library without any external dependencies. Can you see if you can use javax.xml.bind.DatatypeConverter (see https://stackoverflow.com/questions/14413169/which-java-library-provides-base64-encoding-decoding)

Reply:
I started with DatatypeConverter but there were couple tricky bits with it. It does not support omitting padding and url safe encoding, when second one can be done, first is a bit of a issue.